pkg/cmd/api/api_test.go

```diff
@@ -355,6 +356,20 @@ func Test_NewCmdApi(t *testing.T) {
 	}
 }
 
+func Test_NewCmdApi_WindowsAbsPath(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.SkipNow()
+	}
+
+	cmd := NewCmdApi(&cmdutil.Factory{}, func(opts *ApiOptions) error {
+		return nil
+	})
+
+	cmd.SetArgs([]string{"C:\\users\\repos"})
```

Comment:
Protip: you can use Go string literals with backticks so that you don't need to escape backslashes, making Windows paths more readable
```suggestion
	cmd.SetArgs([]string{`C:\users\repos`})
```

Reply:
Great tip, @mislav! I did not know that.